test/lib/ufe/testReferenceCommands.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python 
+
+import fixturesUtils
+from maya import cmds
+from maya import standalone
+import mayaUsd.ufe
+import mayaUtils
+import usdUfe
+import ufe
+import unittest
+import usdUtils
+import testUtils
+
+#####################################################################
+#
+# Helper to compare strings.
+
+def filterUsdStr(usdSceneStr):
+    '''Remove empty lines and lines starting with pound character.'''
+    nonBlankLines = filter(None, [l.strip() for l in usdSceneStr.splitlines()])
+    finalLines = [l for l in nonBlankLines if not l.startswith('#')]
+    return '\n'.join(finalLines)
```

Comment:
Is this identical to the one above and below? If so can it be moved to usdUtils.py?